DS/Assignment1/DoublyNode.java

```diff
@@ -0,0 +1,42 @@
+package datastructureclasses;
+
+public class DoublyNode {
+    private int data;
+    private DoublyNode next, prev;
+
+    public DoublyNode() {
+	next = null;
+	prev = null;
+	data = 0;
+    }
+
+    public DoublyNode(int d, DoublyNode n, DoublyNode p) {
```

Comment:
No need of 2nd and 3rd parameter, as you always pass the null. and variables are by-default null.